cluster.yml

```diff
@@ -26,6 +26,19 @@
     - { role: kubespray-defaults}
     - { role: bootstrap-os, tags: bootstrap-os}
 
+- hosts: k8s-cluster:etcd:calico-rr
```

Comment:
This play takes several minutes because of delegate_to on large clusters. This should be disabled by default and only enabled if someone needs it.

Reply:
Then you can't run kubespray a import_playbook like stated in the docs.

Reply:
But maybe we just need that for etcd and not calico-rr, I am not so deep in the kuebspray roles. At least etcd is broken without that.